tests/test_git_repo_scanner.py

```diff
@@ -98,18 +98,14 @@ def test_extra_exclusions_get_added(self, mock_load: mock.MagicMock):
     def test_extra_signatures_get_added(self, mock_load: mock.MagicMock):
         mock_load.return_value = (
             self.data_dir / "pyproject.toml",
-            {
-                "exclude_signatures": ["foo", "bar"],
-                "exclude_findings": [{"signature": "foo/bar", "reason": "test reason"}],
-            },
+            {"exclude_signatures": ["foo", "bar"]},
         )
+        self.global_options.exclude_signatures = ("foo", "bar")
```

Comment:
```suggestion
        self.global_options.exclude_signatures = ("bar")
```
This way we pull just one of the exclusions from each source, and ensure they are being joined together.